src/libs/ReportUtils.ts

```diff
@@ -5247,7 +5219,7 @@ function getAllAncestorReportActions(report: Report | null | undefined, shouldHi
     return allAncestors.reverse();
 }
 
-function getAllAncestorReportActionIDs(report: Report | null | undefined): AncestorIDs {
+function getAllAncestorReportActionIDs(report: Report | null | undefined, includeTransactionThread: boolean = false): AncestorIDs {
     if (!report) {
```

Comment:
I added `includeTransactionThread` so the money request report replies count is updated too. This is to fix the failed unit test too.

Reply:
The other usage of `getAllAncestorReportActionIDs` is to show the thread ancestor levels (idk what it's called) as shown below:
<img width="808" alt="image" src="https://github.com/Expensify/App/assets/50919443/e728800b-bd83-4bfd-80ab-3eaa9adf8c68">

but we don't want to show it when we are on a transaction thread.
<img width="806" alt="image" src="https://github.com/Expensify/App/assets/50919443/0c7af32c-a4aa-4e64-ae86-c8c2aab42efc">

Reply:
Ahh I see.